tests/test_merge.py

```diff
@@ -43,6 +44,41 @@ def take_first(conflict, _, __, ___):
         self.assertEqual(m.unified_patches,
                          [('change', 'changeme', ('Jo', 'Joe'))])
 
+    def test_continue_run_multiple_conflicts_per_patch(self):
+        lca = {'foo': [{'x': [1, 2, 3]}, {'y': [2, 3, 4]}]}
```

Comment:
> Also, when regression testing the bug it made sense to have two separate tests for each case.
@mihaibivol since we are using `pytest`, you can use `@pytest.mark.parameterize` and th